README.md

```diff
@@ -33,6 +33,8 @@ In order to build the installer, you'll need to install some tools.
     npm install
     ```
 
+If either of the above steps fail, try deleting the c:\Users\<username>\.electron folder.
```

Comment:
Note that I was getting strange errors trying to run builds and couldn't re-bootstrap the npm install. Found this:

https://github.com/electron-userland/electron-prebuilt/issues/96

And lo, I could build again.